modules/apps/digital-signature/digital-signature-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -155,4 +155,18 @@
 			max-height: 100%;
 		}
 	}
+
+	.file-entries-list .has-error {
+		background-color: #feefef;
+		border: 1px solid #f48989;
+
+		.quick-action-menu {
+			background-color: #feefef;
+		}
+
+		.text-error {
```

Comment:
This class should be placed before `.quick-action-menu`

To respect the alphabetical order